dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -160,4 +161,74 @@ public final Response getRecentBaseTypes(@Context final HttpServletRequest reque
 
     return response;
   } // getTypes.
+
+  /**
+   * Return a list of {@link ContentType}, entity response syntax:.
+   *
+   * <code>
+   *  {
+   *      contentTypes: array of ContentType
+   *      total: total number of content types
+   *  }
+   * <code/>
+   *
+   * Url sintax: contenttype/query/query-string/limit/n-limit/offset/n-offset/orderby/fieldname-order_direction
```

Comment:
http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api#advanced-queries